go.mod

```
@@ -21,9 +23,10 @@ require (
 	github.com/cpuguy83/dockercfg v0.3.2 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/distribution/reference v0.6.0 // indirect
-	github.com/docker/docker v27.5.1+incompatible // indirect
+	github.com/docker/docker v28.0.4+incompatible // indirect
```

```suggestion
	github.com/docker/docker v27.5.1+incompatible // indirect
```

Don't bump to Docker v28. That causes issues.